src/components/Instructors.tsx

```diff
@@ -102,11 +103,37 @@ export const instructors: Array<Instructor> = [
     website: "https://chrissyalbert.github.io/",
     twitter: "",
     image: chrissy
+  },
+  {
+    key: 6,
+    title: "Alex Maskovyak",
+    specialty: "Full Stack, DevOps",
+    text: `Software design has always been my passion. "Computers" were my destiny from an early age.
+
+    Come time, come age, come solar flare: nothing will corrupt the memory of my first programming book. That tome wasn't only a present from my grandparents—it was a future. Its pages back then held me absolutely spellbound. I spent hours laboriously reciting line after line of its cryptic script on my keyboard; certain that when the ritual was complete that the incantation would conjure images to life.
+    
+    Those 8-bit graphics wouldn't impress an eight year-old today, but they completely enchanted the apprentice coder that I was then. The sense of wonder from those days remains with me. It’s what’s held me captivated for the last 15 years of professional development experience. Bringing that sort of enchantment to the lives of others is what inspires me today. The magic of programming is that the tools of the craft are empowering
+    
+    Welcome to SolveStack. I hope it casts the same spell on you!
+    `,
+    linkedIn: "https://www.linkedin.com/in/alexmaskovyak/",
+    website: "",
+    twitter: "https://twitter.com/AlexMaskovyak",
+    image: alex
   }
 ];
 
 export const InstructorSquare: FunctionComponent<Instructor> = (props) => {
   const classes = useStyles();
+  
+  /* respect line breaks and construct elements */
```

Comment:
Nice! I didn't know how to do this. Thank you for leading the way!

Reply:
Not a problem! But this does show how scope creep can take a 10 minutes problem and make it a 40 minute one. Engineers...we do it to ourselves.

I do want to break this out into a separate function and possible wrap it as a new component. But that's so beyond the need right now.